bip-0141.mediawiki

```diff
@@ -116,6 +116,16 @@ Blocks are currently limited to 1,000,000 bytes (1MB) total size. We change this
 
 The new rule is ''block weight'' ≤ 4,000,000.
 
+===== Transaction size calculations =====
+
```

Comment:
I think your language here is fine, but since these terms aren't used in the consensus limits anywhere, my only nit would be to consider moving them to a new section ("Additional definitions", maybe?).